crates/stark-backend/src/rap.rs

```diff
@@ -19,6 +19,8 @@ pub trait BaseAirWithPublicValues<F>: BaseAir<F> {
     fn num_public_values(&self) -> usize {
         0
     }
+
+    fn columns(&self) -> Vec<String>;
```

Comment:
Added `columns` to BaseAirWithPublicValues, because that allows us to have it in any of the airs we get (which I understand must implement at least `Air` and `BaseAirWithPublicValues` and `PartitionedBaseAir`).

Recommendations on better places/ways to add it, are welcome.